MANIFEST.in

```diff
@@ -3,3 +3,17 @@ include README.md
 
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
+
+recursive-include curlylint *.json
+recursive-include curlylint *.typed
+
+recursive-include docs *.md
+
+recursive-include stubs *.pyi
+
+recursive-include tests *.html
+recursive-include tests *.liquid
+recursive-include tests *.md
+recursive-include tests *.njk
+recursive-include tests *.toml
+recursive-include tests *.twig
```

Comment:
Currently this `tests` folder only contains sample templates I use when manually testing specific patterns. It’s a few megabytes so I’ll remove this for now.

```suggestion
```